core-web/apps/mcp-server/README.md

```diff
@@ -398,6 +399,34 @@ The MCP server includes comprehensive logging:
 -   **Request/Response Logging**: Full API interaction logging in verbose mode
 -   **Performance Monitoring**: Request timing and performance metrics
 
+### Response Truncation
+
+The MCP server supports configurable response truncation to manage large response sizes:
+
+-   **Environment Control**: Set `MCP_RESPONSE_MAX_LENGTH` to control maximum response length
+-   **Default Behavior**: When not set, responses are returned in full (no truncation)
+-   **Flexible Configuration**: Different deployment scenarios can use different limits
+-   **Truncation Logging**: When truncation occurs, it's logged with original and truncated lengths
+-   **Backwards Compatibility**: Existing deployments continue working without changes
+
+**Example configurations:**
+
+```bash
+# No truncation (default)
+# MCP_RESPONSE_MAX_LENGTH is not set
+
+# Limit responses to 1000 characters
+export MCP_RESPONSE_MAX_LENGTH=1000
+
+# Limit responses to 5000 characters for high-capacity clients
+export MCP_RESPONSE_MAX_LENGTH=5000
+```
```

Comment:
We have examples of MCP server config, there is where we have to add the examples.